README.md

```diff
@@ -7,6 +7,8 @@
 [![License: MIT][mit]](https://opensource.org/licenses/MIT)
 [![open-sir][circleci]](https://circleci.com/gh/open-sir/open-sir)
 [![Code style: black][black]](https://github.com/psf/black)
+[![Documentation Status](https://readthedocs.org/projects/open-sir/badge/?version=latest)](https://open-sir.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Maybe we can use an alias for this URL, like the ones defined at the top of this readme file (it makes reading the markdown defining the badge easier)

Reply:
Sure, much better